src/sagemaker/tensorflow/defaults.py

```diff
@@ -12,4 +12,4 @@
 # language governing permissions and limitations under the License.
 from __future__ import absolute_import
 
-TF_VERSION = '1.11'
```

Comment:
Don't change default version.

Add LATEST_VERSION to the estimator instead, e.g.: https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/mxnet/estimator.py#L33

Reply:
For Mxnet also the default version in defaults.py and LATEST_VERSION in estimator.py is explicityly defined. 
MxNet estimatopr.py => https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/mxnet/estimator.py#L33
MxNet defaults.py -> https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/mxnet/defaults.py

Reply:
Don't change the version @uditbhatia . Version is a required field starting by TF 1.12.

Reply:
I didnt changed any field